cityofaustin/atd-vz-data
# 9995 refactor notes component

## Associated issues
Closes cityofaustin/atd-data-tech/issues/9995

This PR refactors Location Notes & Crash Notes to use the same reusable Notes component. Prior to this PR there were separate Notes components for Crash Notes and Location Notes which was unnecessary because they were doing the exact same thing aside from using different variables. The trick was finding a way for the query/mutation calls in the new Notes component to be reusable for both Crashes and Locations despite having slightly different variables (the location_notes table uses a location_id while crash_notes uses a crash_id).

## Testing
**URL to test:** 
https://deploy-preview-1107--atd-vze-staging.netlify.app/#/dashboard


**Steps to test:**
1. Go to any crash details page.
2. Add a note, edit it, cancel, edit again, save, then delete the note. Ensure it's all working as expected.
3. Repeat for any location details page.

---
#### Ship list
